src/TodoListItem.js

```diff
@@ -1,11 +1,13 @@
-import React from "react";
+import React from 'react'
 
+function TodoListItem({todo,onRemoveTodo}) {
+    console.log({todo})
+    return (
+        <div>
+           <li key={todo.id}>{todo.title}<button  onClick={()=>onRemoveTodo(todo.id)} type="button">Remove</button> </li> 
```

Comment:
This line is getting to be really long and can be broken up into multiple lines such as: 

list item opening tag
-- new indentation --
{todo.title}
button opening tag
-- new indentation --
Remove
button closing tag (matching the opening tag indentation)
list item closing tag (matching the closing tag indentation)